app.js

```diff
@@ -9,74 +9,96 @@ import {
   createKboOrganisation,
```

Comment:
**Remark:** The function `createKboOrganisation` is not defined or used, it seems to have been replaced by `createNewKboOrg`.